webapps/ROOT/index.jsp

```diff
@@ -25,11 +25,11 @@ request.setAttribute("tomcatExamplesUrl", "/examples/");
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8" />
+        <meta charset="UTF-8">
         <title><%=request.getServletContext().getServerInfo() %></title>
-        <link href="favicon.ico" rel="icon" type="image/x-icon" />
-        <link href="favicon.ico" rel="shortcut icon" type="image/x-icon" />
-        <link href="tomcat.css" rel="stylesheet" type="text/css" />
+        <link href="favicon.ico" rel="icon" type="image/x-icon">
```

Comment:
in fact we can remove the line but the favicon.ico will be anyway requested by a browser. The tag makes sense when the file name is another